test.js

```diff
@@ -2,16 +2,16 @@
 
 'use strict'
 
-var shell = require('shelljs')
-var fs = require('fs')
-var path = require('path')
-var stream = require('stream')
-var mockGit = require('mock-git')
-var mockery = require('mockery')
-var semver = require('semver')
-var formatCommitMessage = require('./lib/format-commit-message')
-var cli = require('./command')
-var standardVersion = require('./index')
+const shell = require('shelljs')
+const fs = require('fs')
+const path = require('path')
+const stream = require('stream')
+const mockGit = require('mock-git')
+const mockery = require('mockery')
+const semver = require('semver')
+const formatCommitMessage = require('./lib/format-commit-message')
+const cli = require('./command')
+const standardVersion = require('./index')
```

Comment:
These lines seem unrelated to this particular change. Could they be pulled out into a separate pull request that refactors the code base into modern ES6+?

Reply:
point taken re: not making more changes than necessary while adding a feature, at the same time since I pulled in a new dependency I usually tend to switch `var` to `const` for the surrounding imports (my feeling being it's better to eliminate this potential vector for bugs, rather than introduce a new vector).

I created a follow up ticket, to go through the codebase more thoroughly -- this feels like a great task for first time contributors:

https://github.com/conventional-changelog/standard-version/issues/276